hts-ft/script-hts-ft.js

```diff
@@ -36,7 +36,9 @@ async function scriptHtsFungibleToken() {
     client = Client.forTestnet().setOperator(operatorId, operatorKey);
     logger.log('Using account:', operatorIdStr);
 
-    //ToDo: Generate a new key for the token's adminKey
+    //Generate a new key for the token's adminKey
+    const adminPrivateKey = PrivateKey.generateECDSA();
+    const adminPublicKey = adminPrivateKey.publicKey;
```

Comment:
Is there a need for a separate admin key for this token in a hello world sequence?
If this is something that can be moved to a tutorial instead, we can avoid the additional complexity.

Reply:
The reason I added the admin key is to show that this key can be completely separate from the the account that responsible for paying for transactions fees and the treasury account key. In the past, when all of these keys are set as the same key, people get confused with the signing requirements and not understanding that if the transaction fee payer account (operator account), designated treasury account and admin key are different then the transaction requires 3 signatures and not 1. 

This step is not required if the operator key is the same as the transaction fee payer key https://github.com/hedera-dev/hello-future-world-js/blob/main/hts-ft/script-hts-ft.js#L72 because on execute the operator key automatically signs all transactions under the hood here https://github.com/hedera-dev/hello-future-world-js/blob/main/hts-ft/script-hts-ft.js#L77.

Setting the admin key is not required. If we want to reduce complexity we can remove that from being set and this example can demonstrate creating an immutable fungible token which cannot update any token properties.

Reply:
removed the use of the admin key